mlflow/langchain/__init__.py

```diff
@@ -593,6 +589,8 @@ def _save_model(model, path, loader_fn, persist_dir):
 
 
 def _load_model(local_model_path, flavor_conf):
+    from mlflow.langchain._langchain_autolog import _update_langchain_model_config
```

Comment:
note: avoid global import so we can use type hint (without worrying about type hint madness with sphinx) in `_langchain_autolog.py`.